client/my-sites/comments/comment-list/index.jsx

```diff
@@ -214,6 +187,86 @@ export class CommentList extends Component {
 	}
 }
 
+export const CommentFaker = WrappedCommentList => class extends Component {
```

Comment:
Let's move this into it's own file. Components/HOC are easier to find when we have one export per file.

Some jsDoc here for intent would be great too, since we don't have any faker patterns in the codebase yet.